src/views/Home.tsx

```diff
@@ -0,0 +1,47 @@
+import { INPUT_VALUE_KEY } from 'appConstants';
+import { Cards, SearchBar } from 'components';
+import { fetchDetailedPokemons } from 'utils';
+import React, { ChangeEvent } from 'react';
+import { PokemonDetails } from 'types';
+import * as Styled from './styled';
+
```

Comment:
so, it is src/modules/SearchPage/index.tsx

Reply:
done